### ISSUE TITLE
timeout on long running recognize

### ISSUE THREAD

**Issue by happysalada** [jabber-tools/google-cognitive-apis#7]
making a long running recognize call will result in an http timeout. (I think that is expected). There is an `operations` api to retrieve pending operations. Are you thinking about implementing that api as well ?

**Comment by adambezecny** [comment]
hi,

this is already implemented, have a look [here](https://github.com/jabber-tools/google-cognitive-apis/blob/main/examples/recognizer_async.rs)

especially these lines are important (notice **llo_reponse** variable): https://github.com/jabber-tools/google-cognitive-apis/blob/main/examples/recognizer_async.rs#L49-L63

Please let me know whether this is what you are after.

regards,

Adam

**Comment by happysalada** [comment]
I used exactly the example.

I was thinking of making the http request on a server and getting the response synchronously. That's when I ran in a timeout since this request takes longer than 30 seconds.

I guess there is nothing to do here, it should just be understood that those long running requests are just running on the server and shouldn't be attempted to be returned in a single http request.
